integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -307,6 +324,277 @@ public void testUpdateDataSourceInDomainUsingAuxiliaryImage() {
     logger.info("Found the DataResource configuration");
   }
 
+  /**
+   * Negative Test to create domain with mismatch mount path in auxiliary image and auxiliaryImageVolumes.
+   * in auxiliaryImageVolumes, set mountPath to "/errorpath"
+   * in auxiliary image, set AUXILIARY_IMAGE_PATH to "/auxiliary"
```

Comment:
Also mention that we check the event for associated warning message.

Reply:
fixed